cmd/rofl/mgmt.go

```diff
@@ -117,21 +117,60 @@ var (
 				cobra.CheckErr(fmt.Errorf("failed to write manifest: %w", err))
 			}
 
-			// Initialize git repository if not already present.
-			_, err = os.Stat(".git")
-			if err != nil && errors.Is(err, os.ErrNotExist) {
-				// Git directory doesn't exist, try to initialize.
-				gitInitCmd := exec.Command("git", "init")
-				if err = gitInitCmd.Run(); err != nil {
-					fmt.Printf("Git repository not initialized: %s.\n", err)
+			// Save current working directory.
```

Comment:
Since you rely on the `git` command for initializing the repo, I see no reason why not simply execute `git rev-parse` to check whether the project folder is already part of the git repo and if not, run `init`?

Reply:
That was my initial idea, but then realized that we need to modify the .gitignore file and need its path anyway.

Reply:
Btw, we already have a "get me the Git root" as we need it when determining the project base dir for use with the Docker builder. Please take a look at `build/env/base.go` which should probably be used instead of reimplementing it.